packages/functions/src/api/service.ts

```diff
@@ -110,7 +110,7 @@ export class Service implements FirebaseFunctions, FirebaseService {
    * @param origin The origin of the local emulator, such as
    * "http://localhost:5005".
    */
-  useFunctionsEmulator(origin: string) {
+  useFunctionsEmulator(origin: string | null) {
```

Comment:
Don't think this function is intended to reset the emulator. Plus it's an API change. I would just leave it to only accept string.

Reply:
Ok, got rid of the test line that uses null to reset this.  Doesn't seem necessary.